app/controllers/contacts_controller.rb

```diff
@@ -5,20 +5,24 @@ class ContactsController < ApplicationController
   skip_verify_authorized
 
   def new
-    @contact_form = ContactForm.new
+    @contact_form = ContactForm.new(welcome: params[:welcome])
+
+    render :new, layout: false
   end
 
   def create
     contact_form = ContactForm.new(contact_form_params)
 
     ContactsMailer.with(contact_form.attributes.symbolize_keys).jira_email.deliver_later
 
-    redirect_to success_contacts_path
+    @welcome = contact_form.welcome?
+
+    render :success, status: :created
```

Comment:
Slight concern that this won't work with turbo. Did you double check that this existing contact modal still works?

Reply:
Yep, already tested that both forms work as intended